wazuh/wazuh-kibana-app
# Bug/3296 cdb list view not working with ipv6

Hi guys,
We have fixed that in the `key` of the `CDB Lists` we can enter ipv6 in this way: `"dead:beef::0:0:0:0:0"`

**To test it:**

Enter `Management/CBD Lists`, create a new one and add it as key `"dead:beef::0:0:0:0:0"` and as value any other. When restarting the cluster and re-entering that list, the key should appear as we have added it.

**Precondition 1**, CBD list information. key: `"dead:beef::0:0:0:0:0"`, value: `"test"`.

- **Given** the browser is logged in the Wazuh kibana app
- **When** the user navigates to CBD lists
- **And** adds a new CBD list entry with the information stated on precondition 1
- **Then** a row will be displayed with the columns key and value correctly displayed

Close: #3296
